src/frontend/package.json

```diff
@@ -26,8 +26,10 @@
         "react-aria-components": "^1.10.1",
         "react-dom": "^19.1.0",
         "react-router": "^7.6.2",
+        "tailwind-variants": "^1.0.0",
```

Comment:
I haven't used tailwind-variants before but this is interesting and seems really helpful. Interesting that it's made by [HeroUI](https://www.heroui.com/?utm_source=tailwind-variants.org) which is one of the component libraries we thought about using and is react-aria + tailwind under the hood!